src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/EmbeddedCompletionContext.cs

```diff
@@ -35,10 +35,10 @@ internal partial class DateAndTimeEmbeddedCompletionProvider
                 _replacementSpan = TextSpan.FromBounds(startPosition, context.Position);
             }
 
-            private static DateTime s_exampleDateTime = DateTime.Parse("2009-06-15T13:45:30.1234567Z");
-            private static CultureInfo s_enUsCulture = CultureInfo.GetCultureInfo("en-US");
-            private static CultureInfo s_primaryCulture = CultureInfo.CurrentCulture;
-            private static CultureInfo? s_secondaryCulture = s_primaryCulture.Equals(s_enUsCulture) ? null : s_enUsCulture;
+            private static readonly DateTime s_exampleDateTime = DateTime.Parse("2009-06-15T13:45:30.1234567Z");
+            private static readonly CultureInfo s_enUsCulture = CultureInfo.GetCultureInfo("en-US");
+            private static readonly CultureInfo s_primaryCulture = CultureInfo.CurrentCulture;
+            private static readonly CultureInfo? s_secondaryCulture = s_primaryCulture.Equals(s_enUsCulture) ? null : s_enUsCulture;
```

Comment:
this conflicts with my latest PR :)

Reply:
Thanks, I'll merge latest and fix these up.